src/dispatch/incident/flows.py

```diff
@@ -635,7 +635,7 @@ def incident_create_closed_flow(
 
 
 @background_task
-def incident_create_flow(*, organization_slug: str, incident_id: int, db_session=None):
+def incident_create_flow(*, incident_id: int, db_session=None):
```

Comment:
Hmm did we test all the calls to this function after removing `organization_slug`?

Reply:
hmm, I think you're right let me add it back.